content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,146 @@ schema: olm.channel
 
+## Composite Template
+A `composite template` can help an operator author manage FBC contributions to multiple catalogs. The template functionality is composed of schemas which represent the author's role and the catalog maintainer's role, and rendering the template performs an explicit negotiation between them.
+
+### Usage
+```sh
+opm alpha render-template composite [flags]
+```
+
+| Flag                | Description                                                                            |
+| ------------------- | -------------------------------------------------------------------------------------- |
+| -f, --catalog-config string | File to use as the catalog configuration file (default "catalogs.yaml")        |
+| -c, --composite-config string | File to use as the composite configuration file (default "catalog/config.yaml") |
+| -h, --help          | help for composite                                                                     |
+| -o, --output string | Output format (json|yaml) (default "json")                                             |
+| --validate          | whether or not the created FBC should be validated (i.e 'opm validate') (default true) |
+| --skip-tls-verify   | skip TLS certificate verification for container image registries while pulling bundles |
+| --use-http          | use plain HTTP for container image registries while pulling bundles                    |
+
+
+
+
+### Specifications
+The `composite template` is composed of two schemas that represent the operator author's role and the catalog maintainer's role.
+
+#### Operator Author
+Representing the operator author is the `olm.composite` schema which defines:
+- Where each input (`Component`) exists
+- How each contribution is mapped to a destination catalog (`Component.Name`). All destination catalogs defined in this file **must** also exist in the catalog configuration file created by the catalog maintainers
+- How each `Component` is processed to generate a catalog contribution (`Component.Strategy`)
+- The directory structure the catalog contribution generation should follow (`Component.Destination`)
+
+The cue schema for the `olm.composite` schema is:
+```cue
+#CompositeConfig: {
+	Schema: "olm.composite"
+	Components: [...#Component]
+}
+
+#Component: {
+	Name:        string
+	Destination: #ComponentDestination
+	Strategy:  #BuildStrategy
+}
+
+#ComponentDestination {
+	Path: string
+}
+
+#BuildStrategy: {
+	Name:   string
+	Template: #TemplateDefinition
+}
+
+#TemplateDefinition: {
+	Schema: string
+}
+```
+
+#### Catalog Maintainer
+Representing the catalog maintainer is the `olm.composite.catalogs` schema which defines:
```

Comment:
Like above:
The `olm.composite.catalogs` schema represents the catalog maintainer role and defines the following resources:

Think resources is the wrong term in both cases. Perhaps, we can use _the following_ as a noun.

Reply:
Addressed in https://github.com/operator-framework/olm-docs/pull/286/commits/1cdb364fe5c9f65cbc47ff7effdb3c41bf104cc0

Reply:
/resolve